desktop-themes/utterly-nord-plasma/spec

```diff
@@ -0,0 +1,3 @@
+VER=3.2+git20240505
+SRCS="git::commit=e513b4dfeddd587a34bfdd9ba6b1d1eac8ecadf5::https://github.com/HimDek/Utterly-Nord-Plasma.git"
```

Comment:
Maybe just set `SUBDIR=.` so you could just copy the whole directory to the target directory?

Reply:
The paths to which files need to be copied are complex and varied, which doesn't significantly streamline the build.